pyecharts/conf.py

```diff
@@ -68,7 +67,8 @@ def get_current_jshost_for_jupyter(self, jshost=None):
         :param jshost:
         """
         jshost = jshost or self.jshost
-        if jshost == SCRIPT_LOCAL_JSHOST:
+        if jshost is None or jshost == SCRIPT_LOCAL_JSHOST:
```

Comment:
SCRIPT_LOCAL_JSHOST is local path to js folder. It wont work under jupyter environment, so I think the suffix '_JSHOST' is misleading.